src/python/pants/backend/scala/dependency_inference/scala_parser.py

```diff
@@ -305,27 +294,25 @@ async def setup_scala_parser_classfiles(jdk: InternalJdk) -> ScalaParserCompiled
                         Coordinate(
                             group="org.scala-lang",
                             artifact="scala-compiler",
-                            version=PARSER_SCALA_VERSION,
+                            version="2.13.8",
```

Comment:
Would be good to preserve this constant probably? It's also used in the `default_artifacts`.

Reply:
Will do.